### PR TITLE
Updated readme with linux info

### PR DESCRIPTION
**Pull request by apainintheneck**
I took a stab at updating the readme to reflect the Linux systemctl support. This is related to #459.

### CODE REVIEW

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Looks good to me, thanks @apainintheneck !

**Review by MikeMcQuaid — APPROVED** [review]
Thanks again @apainintheneck, great work!

File: README.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
 # Homebrew Services
 
-Manage background services with macOS' `launchctl` daemon manager.
+Manage background services using the daemon manager `launchctl` on macOS' or `systemctl` on Linux.
```
Comment (Rylan12) [inline README.md:3]: I don't think the `'` after macOS makes sense anymore, right?

Comment (apainintheneck) [reply README.md:3]: Yeah, you're right. That doesn't make sense anymore.

**Pull request merged**